### ISSUE TITLE
Add Discord Presence and InnerTune Audio Normalization features

### ISSUE THREAD

**Issue by rajinstar** [OuterTune/OuterTune#117]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

1. InnerTune has a feature called Discord Presence. Which will help me to show my Discord friends which song I currently play, and also it will help you to promote your app.
 
2. Convert OuterTune Audio Normalization to InnerTune Audio Normalization. Which means, please change OuterTune Audio Normalization features and add the same to the same InnerTune Audio Normalization. I thought InnerTune Audio Normalization was far better and enhanced the music experience. 

3. Add InnerTune's some features and redesign the app and make it beautiful. Redesigning and making it beautiful is optional. But please add InnerTune's features. 
 
That is how this app will become unbeatable!

### Why do you want this feature?

1. When I play music, I like to showcase it to my friends. Like, which song do I currently play right now? So that my Discord friends will know. The problem I face is that I want to let my friends know which song I currently play. 
 
2. Audio normalization features are very important. InterTune's Audio Normalization is better. But OuterTune Audio Normalization is not up to the mark. When turning on songs, the audio level doesn't change at all. Just like the audio normalization turned on but no effect. That's the problem I face while using this app. 
 
 
Finally, overall, the app is the best. Just need to fix it and add some features, and that's it.

### Additional information

Please, I request you please introduce these two features to OuterTune. These features will enhance my music listening experience.

**Comment by gechoto** [comment]
> Convert OuterTune Audio Normalization to InnerTune Audio Normalization.

The "Normalization" in the latest InnerTune version is broken and not normalization anymore. It was reverted so it will be back at using the old normalization method in the next InnerTune version.

Converting the current normalization to the broken one from the latest InnerTune release would be a bad idea since it introduces compression and destroys the dynamics of songs.

If the feature you mean would be added it should be a separate toggle - maybe called "Loudness enhancement" or something with "compression" in the name because this is NOT normalization.

**Comment by rajinstar** [comment]
You are right actually. Ok whatever, please add Discord Rich Presence just like InnerTune

**Comment by MrT-tak** [comment]
> You are right actually. Ok whatever, please add Discord Rich Presence just like InnerTune 

@gechoto ETA for Discord Rich Presence?
Is there issues with that as well?

**Comment by MrT-tak** [comment]
Is this a hard fork?
